src/Evolution/Systems/RadiationTransport/M1Grey/Characteristics.hpp

```diff
@@ -0,0 +1,69 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/RadiationTransport/M1Grey/Tags.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"  // IWYU pragma: keep
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+/// \endcond
+
+namespace RadiationTransport {
+namespace M1Grey {
+
+// @{
+/*!
+ * \brief Compute the characteristic speeds for the M1 system
+ *
+ * At this point, for testing purposes, we just set all
```

Comment:
Could you please file an issue for this?

Reply:
Filed #1709